backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/sort/impl/LuceneTextFieldSortBuilder.java

```diff
@@ -36,7 +37,14 @@ protected Object encodeMissingAs(F converted) {
 	public void buildAndContribute(LuceneSearchSortCollector collector) {
 		// For STRING type, missing value must be either STRING_FIRST or STRING_LAST.
 		// Otherwise we need a CUSTOM type.
```

Comment:
You should probably move this comment to `getSortField()`

Reply:
I squashed both to `HSEARCH-2254 Support sort on nested field - Lucene`.